src/library/scala/PartialFunction.scala

```diff
@@ -86,6 +86,30 @@ trait PartialFunction[-A, +B] extends (A => B) { self =>
   override def andThen[C](k: B => C): PartialFunction[A, C] =
     new AndThen[A, B, C] (this, k)
 
+  /**
+    * Composes this partial function with other partial function that
+    * gets applied to results of this partial function.
```

Comment:
Something about this description (as well as the one for `compose` seems confusing to me; I think it could use a clearer wording. I can't think of anything off the top of my head, but I will think on it.